code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -984,6 +988,26 @@
 		return stored_files
 	return stored_files + inserted_disk.stored_files
 
+/// Returns how relevant the current security level is:
+#define ALERT_RELEVANCY_SAFE 0 /// * 0: User is not in immediate danger and not needed for some station-critical task.
+#define ALERT_RELEVANCY_WARN 1 /// * 1: Danger is around, but the user is not directly needed to handle it.
+#define ALERT_RELEVANCY_PERTINENT 2/// * 2: Danger is around and the user is responsible for handling it.
```

Comment:
These should be undefined at the end of the proc. The linters is warning you about that.